### DISCUSSION TITLE
How to print the depth at position?

### DISCUSSION THREAD

**Discussion by j-friedl** [ifm/ifm3d#365] [Q&A] [answered]
Hi, I was wondering how to get the distance at a specified point in meters? It would be great if someone could add that also to the docs. :)

Right now I have something like this:
```
    ifm3d::Image distImage = distImageBuffer->DistanceImage();

    for(int i = 0; i < distImage.width() * distImage.height(); i++)
      std::cout << distImage.at<_Float32>(i) << std::endl;
    return 0;
```

A short section of the output:

> 0
> 0
> 0
> 0
> 0
> 0
> 0
> 0
> 0
> 0.435492
> 0.437018
> 0.436866
> 0.431525
> 0.429541
> 0.432898
> 0.459144

**Comment by j-friedl** [comment, marked as answer]
I'm sorry, I was a bit hasty in posting the question. [This post](https://github.com/ifm/ifm3d/issues/333) describes the issue better.

**Reply by inbangsa** [reply]
@j-friedl  you can use code in this comment to print xyz values 

https://github.com/ifm/ifm3d/issues/333#issuecomment-972601221

**Comment by lola-masson** [comment]
Hey @j-friedl,
Looks like you have a lot of questions (which is awesome!). Do you want to jump on a quick phone call so that we can go through your setup and what you're trying to do? I think this would be a quicker way to get you in a good place with the O3R. You can email `support.robotics@ifm.com` and we will setup a virtual meeting.

**Reply by j-friedl** [reply]
@lola-masson thank you for the suggestion, I've written an email to info@ifm.com and they didn't reply so I started to spam you in here (sorry for the many questions btw.). As posted above, I've found the older post and can go from there right now. However, if there'll come up new questions, I'd be happy to get in touch via email, thank you!

**Reply by lola-masson** [reply]
No worries! Let us know what is the best way to assist you. Just to recap:
- All our documentation is [here](https://ifm3d.com/sphinx-doc/build/html/index.html). You will find all the O3R documentation and API examples there.
- For most in depth questions and to setup training, contact `support.robotics@ifm.com` and the sales engineer responsible for your account, if you know who it is.
- Feel free to post issues on github with suggestions or questions as well. We do track these internally.
- Please close the issue once it has been resolved. This helps us track what still need to be answered.
Thanks!
